projects/js-packages/components/changelog/remove-threat-components

```diff
@@ -0,0 +1,4 @@
+Significance: major
+Type: removed
+
+Removed threat components
```

Comment:
Just a suggestion. It seems unlikely there are any reusers who might care, but we can tell them where to look in case there are any.
```suggestion
Removed threat components to `@automattic/jetpack-scan`.
```